packages/nc-gui-v2/components/general/SocialCard.vue

```diff
@@ -1,16 +1,70 @@
 <script setup lang="ts">
 import { enumColor as colors } from '#imports'
 
+const { showSwaggerLink } = defineProps<{ showSwaggerLink?: boolean }>()
```

Comment:
The swagger link isn't really part of our "socials" so instead, why not just use a slot instead of adding it into here?